cmake/ProjUtilities.cmake

```diff
@@ -76,27 +76,42 @@ function(configure_proj_pc)
   set(datadir "$\{datarootdir\}")
   set(PACKAGE "proj")
   set(VERSION ${PROJ_VERSION})
-  # Build list for Libs.private
-  set(EXTRA_LIBS
-    -lstdc++
-    -lsqlite3
-    ${CMAKE_THREAD_LIBS_INIT}
-  )
+  # Build strings of dependencies (Libs.private, Requires.private)
+  set(EXTRA_LIBS "${CMAKE_THREAD_LIBS_INIT}")
+  set(EXTRA_REQUIRES "")
+  option(USE_PKGCONFIG_REQUIRES "Use 'Requires' instead 'Libs' in proj.pc" ON)
+  macro(add_module_or_libs MODULE)
+    if(USE_PKGCONFIG_REQUIRES)
+      list(APPEND EXTRA_REQUIRES "${MODULE}")
+    else()
+      list(APPEND EXTRA_LIBS "${ARGN}")
+    endif()
+  endmacro()
+  add_module_or_libs(sqlite3 -lsqlite3)
   if(TIFF_ENABLED)
-    list(APPEND EXTRA_LIBS -ltiff)
+    add_module_or_libs(libtiff-4 -ltiff -lfoo)
```

Comment:
-lfoo ?

Reply:
Good catch :facepalm:  This was from manual testing with multiple libs.